src/api/todos.ts

```diff
@@ -2,7 +2,17 @@ import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
 export const getTodos = (userId: number) => {
-  return client.get<Todo[]>(`/todos?userId=${userId}`);
+  return client.get<Todo[]>(`/todos?userId=${userId}`) || null;
 };
 
-// Add more methods here
+export const createTodo = (
+  title: string,
+  userId: number,
+  completed = false,
+) => {
```

Comment:
It's better to pass an object, so you don't make mistake with order of the params